src/components/Header.tsx

```diff
@@ -16,15 +16,14 @@ export const Header = ({ children }: HeaderProps) => {
   return (
     <header className="sticky top-0 z-10 flex-none border-b border-black/5 bg-zinc-50/70 backdrop-blur dark:border-white/5 dark:bg-zinc-900/70">
       <div className="flex h-[3.75rem] items-center justify-between space-x-8 px-8">
-        <div className="flex flex-shrink-0 items-center">
-          <Link href="/" className="w-72">
+        <div className="flex flex-auto flex-shrink-0 items-center">
+          <Link href="/" className="w-36">
```

Comment:
```suggestion
          <Link href="/">
```
We don't actually need a width here as this is the only element in the div so it is free to be any size.